packages/media/README.md

```diff
@@ -39,6 +39,30 @@ Images:
 * `.webp`
 </details>
 
+## Extending supported extensions
+
+Media loader exposes its `Set` of file extensions so you can add and remove items from it:
+
+```console
+$ node --import ./register.mts ./main.tsx
+```
+
+```js
+// ./register.mts
+
+import module from 'node:module';
+import { exts } from '@nodejs-loaders/media/media.loader.mjs';
+
+exts.add('.pdf');
+module.register('@nodejs-loaders/media', import.meta.url);
+```
+
+```jsx
+// main.tsx
+
+import somePdf from './some.pdf'; // somePdf = '[…]/some.pdf'
+```
+
```

Comment:
As a user, I was looking for a "Usage" section in the readme on npm

Maybe adding a basic `## Usage` section with the simple usage like in #170 would be ok?

Reply:
Hm, why? I have the feeling that:

1. the basic instructions will not change (or at least not more than 2+ times per year)
2. with a modern code editor, changing 1 file is not much faster than changing 20 files, with search + replace patterns and other refactoring tools

Reply:
Ok, but will it ever change? If there's a high likelihood that the instructions will never change, then the number of releases will still be just N, one single time.

Anyway, just bringing it up as a user who likes the usage instructions to be in the package readme.